src/test/app/forms/models/claimReference.ts

```diff
@@ -51,7 +51,7 @@ describe('ClaimReference', () => {
     })
 
     it('should accept valid CMC claim reference ignoring case', () => {
-      const errors = validator.validateSync(new ClaimReference('000mc001'))
+      const errors = validator.validateSync(new ClaimReference('000MC001'))
```

Comment:
Seems like the test title is no longer valid then?

Reply:
oops. I changed it as part of a bulk update and missed it when I undid it. will change